scripts/docker/build-centos6/Dockerfile.deps

```diff
@@ -28,8 +28,9 @@ RUN rm /etc/yum.repos.d/CentOS-SCLo-scl.repo
 #  - doxygen & JSON.pm
 RUN yum install -y doxygen graphviz perl-JSON
 #  - antora (npm needed)
-RUN curl -sL https://rpm.nodesource.com/setup_8.x | bash -
+RUN curl -sL https://rpm.nodesource.com/setup_10.x | bash -
 RUN yum install -y nodejs
+RUN rm /etc/yum.repos.d/nodesource-*
```

Comment:
It's necessary to bypass an error with the _nodesource_ repo reporting `not found / 404`.